database/migrations/sqls/20240730102034-create-session-table-up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE session (
+    token VARCHAR(255) NOT NULL,
+    user_id INT NOT NULL,
```

Comment:
Make this a foreign key to the users table, you will need to join them to get the `user_type`